XbSymbolDatabase.c

```diff
@@ -1512,13 +1515,14 @@ bool XbSymbolScan(const void* xb_header_addr,
 
                                         XbSymbolOutputMessageFormat(XB_OUTPUT_MESSAGE_DEBUG, "Scanning %.8s library in %.8s section", LibraryStr, SectionName);
 
-                                        XbSymbolScanOOVPA(SymbolDBList[d2].OovpaTable, SymbolDBList[d2].OovpaTableCount, LibraryStr, SymbolDBList[d2].LibSec.library,
+                                        XbSymbolScanOOVPA(SymbolDBList[d2].OovpaTable, SymbolDBList[d2].OovpaTableCount, LibraryStr, LibraryFlag,
                                             pSectionScan, BuildVersion, register_func, xb_start_virt_addr);
                                         break;
                                     }
                                 }
                             }
-                            break;
+                            // We're not using break here since we are combining LTCG flag to std flag's oovpa database such as D3D8.
+                            //break;
```

Comment:
Here, same remark as above (enabling `break` when only one bit is set in `LibraryFlag`)